include/EASTL/sort.h

```diff
@@ -773,10 +794,10 @@ namespace eastl
 			if(*i < *first)
 			{
 				EASTL_VALIDATE_COMPARE(!(*first < *i)); // Validate that the compare function is sane.
-				const value_type temp(*i);
-				*i = *first;
+				value_type temp(eastl::forward<value_type>(*i));
```

Comment:
This is the same issue as before:
if `temp` is of type `const value_type` but `value_type` is only move-assignable, then when `adjust_heap` is invoked then the `temp` cannot be assigned.